frontend/src/hex/ByteclassView.svelte

```diff
@@ -124,17 +125,13 @@
 
     context.strokeStyle = "red";
     context.lineWidth = Math.ceil(canvas.height / 512);
-    if (
-      data !== undefined &&
-      data.length > alignment * 3 &&
-      $scrollY.viewHeight !== 1
-    ) {
+    if (data !== undefined && data.length > alignment * 3) {
       // Offset Y by 0.5 because of: https://stackoverflow.com/a/48970774
       context.strokeRect(
         0,
-        Math.ceil($scrollY.top * canvas.height) - 0.5,
+        Math.ceil((currentPosition / dataLength) * canvas.height) - 0.5,
         alignment,
-        Math.ceil(($scrollY.viewHeight * canvas.height) / 2)
+        Math.ceil(($screenHeight / dataLength) * canvas.height) - 0.5
```

Comment:
Why does this no longer need to be divided by two?

Reply:
Why did it need to in the first place? 
The screen height should be the number of bytes being displayed as it is calculated with:
`  $screenHeight =
      Math.floor(hexDisplay.offsetHeight / lineHeight) * alignment;`
being the number of lines currently being displayed * 16 bytes per line. i.e bytes per line. 

The height of the rectangle is then the percentage of the current data being displayed * the size of the canvas.